### PR TITLE
fixed the test in #125

### PR DESCRIPTION
**Pull request by scottbommarito**
i'm probably gonna have to make some more changes so that the tests are easier to understand (eventually) but this fixes Geoffrey's test case that was failing on Android only because the emulator is slower than the iOS emulator

### CODE REVIEW

**Comment by geof90** [comment]
LGTM!

**Comment by scottbommarito** [comment]
@lostintangent 
i think i need another sign-off lol

**Comment by scottbommarito** [comment]
had to make one more change because the test was not labeled as not a core test

(yea ik it's confusing, will be much clearer soon)

perfect score on VSTS
https://mobiledevops.visualstudio.com/VS%20Client/_build?_a=summary&buildId=4300

File: test/test.ts [hunk 1]
```diff
@@ -862,59 +862,56 @@ function runTests(targetPlatform: platform.IPlatform, useWkWebView: boolean): vo
             }
         });
 
-        describe("#localPackage.installOnNextRestart2x", function() {
+        if (!onlyRunCoreTests) {
```
Comment (dlebu) [inline test/test.ts]: Mocha has the ability to filter tests after name. If all core tests would contain a `core` prefix, you would not need to do this.

**Comment by dlebu** [comment]
LGTM

File: test/test.ts (hunk 1 above)
Comment (scottbommarito) [inline test/test.ts]: interesting...i'm changing the whole implementation pretty soon (with the change that will support react-native testing) but this is good to know

**Pull request merged**